sympy/series/tests/test_series.py

```diff
@@ -213,3 +213,7 @@ def test_issue_15539():
         + O(x**(-6), (x, -oo)))
     assert series(atan(x), x, oo) == (-1/(5*x**5) + 1/(3*x**3) - 1/x + pi/2
         + O(x**(-6), (x, oo)))
+
+
+def test_issue_7259():
+    assert series(LambertW(x), x) == x - x**2 + 3*x**3/2 - 8*x**4/3 + 125*x**5/24 + O(x**6)
```

Comment:
@ArighnaIITG Can you check if ``fps``/ ``aseries`` handles this?

Reply:
No. @leosartaj